src/NumericalAlgorithms/FiniteDifference/HighOrderFluxCorrection.hpp

```diff
@@ -97,7 +99,16 @@ void cartesian_high_order_fluxes_using_nodes(
         ghost_cell_inertial_flux,
     const Mesh<Dim>& subcell_mesh, const size_t number_of_ghost_cells,
     [[maybe_unused]] const std::array<gsl::span<std::uint8_t>, Dim>&
-        reconstruction_order = {}) {
+        reconstruction_order = {},
+    [[maybe_unused]] const bool aligned_coordinates = true,
+    [[maybe_unused]] const std::array<tnsr::i<DataVector, Dim, Frame::Inertial>,
+                                      Dim>& lower_normal = {},
+    [[maybe_unused]] const std::array<tnsr::i<DataVector, Dim, Frame::Inertial>,
+                                      Dim>& upper_normal = {}) {
+  std::array<tnsr::i<DataVector, Dim, Frame::Inertial>, Dim> effective_normal =
+      make_with_value<std::array<tnsr::i<DataVector, Dim>, Dim>>(
+          cell_centered_inertial_flux, 0.0);
```

Comment:
Can we combine the memory allocations here into one large buffer? One possibility is to use a `Variables` with a tag of `ij` and then the you have the 2nd-rank like thing you have here with arrays, but would require changes below. Another is to use a `DataVector buffer{dim*dim*num_pts};` and point into it